src/test/java/com/google/step/datamanager/UserManagerDatastoreTest.java

```diff
@@ -0,0 +1,122 @@
+package com.google.step.datamanager;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.step.model.User;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class UserManagerDatastoreTest {
+
+  private static final String EMAIL_A = "testa@example.com";
+  private static final String EMAIL_B = "testb@example.com";
+
+  private static final String USERNAME_A = "Alice";
+  private static final String USERNAME_B = "Bob";
+
+  private static final String BIO_A = "Hello world.";
+  private static final String BIO_B = "Hello I'm Bob.";
+
+  private static final String BLOBKEY = "a_blob_key";
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig().setApplyAllHighRepJobPolicy());
+
+  private final UserManagerDatastore userManagerDatastore = new UserManagerDatastore();
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void testCreateUser() {
+    User user = userManagerDatastore.createUser(EMAIL_A);
+    Assert.assertEquals(EMAIL_A, user.email);
+    Assert.assertEquals(EMAIL_A, user.username);
+    Assert.assertEquals("", user.bio);
+    Assert.assertFalse(user.photoBlobKey.isPresent());
+  }
+
+  @Test
+  public void testReadUserByEmail_firstTime() {
+    User user = userManagerDatastore.readOrCreateUserByEmail(EMAIL_B);
+    Assert.assertEquals(EMAIL_B, user.email);
```

Comment:
Just use a static import for `assertEquals` and you can get rid of `Assert.` everywhere in this file.